src/wallets/IdentityWallet.ts

```diff
@@ -225,4 +317,27 @@ export class IdentityWallet implements TLWallet {
       )
     }
   }
+
+  private verifyFromField(rawTx: RawTxObject) {
+    if (!(rawTx.from === this.address)) {
+      throw new Error(
+        `The from field of the meta-transaction has to match with the address of the identity, from: ${
+          rawTx.from
+        }`
+      )
+    }
+  }
+
+  private buildMetaTransaction(rawTx: RawTxObject): MetaTransaction {
+    const metaTransaction: MetaTransaction = {
+      data: ethers.utils.solidityKeccak256(['bytes'], [rawTx.data]),
+      extraData: '0x',
+      from: rawTx.from,
+      nonce: rawTx.nonce.toString(),
```

Comment:
why does that have to be a string?

Reply:
I can see in the relay server that it is interpreted in the `MetaTransactionSchema` as the same type as `value` (BigInteger) which afaict should be a string.

Reply:
Ah I am always confused that some of these types are types of the api. I'll create an issue to make that explicit.